multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/MarkdownDefaults.kt

```diff
@@ -1,110 +0,0 @@
-package com.mikepenz.markdown
-
-import androidx.compose.material.MaterialTheme
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.Immutable
-import androidx.compose.runtime.Stable
-import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.text.TextStyle
-import org.intellij.markdown.IElementType
-import org.intellij.markdown.MarkdownElementTypes
-
-
-/**
- * Contains the default values used by [Markdown].
- */
-object MarkdownDefaults {
```

Comment:
Could you please outline the choice of dropping the defaults implementation - requiring every usecase to provide their expected styling?